xml/Microsoft.Build.Framework/SdkResult.xml

```diff
@@ -14,13 +14,17 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>
-                An abstract interface class to indicate SDK resolver success or failure.
-                <remarks>
```

Comment:
@joelmartinez It seems like a build warning could/should be generated if `<remarks>` tags are found within `<summary>` or `<returns>` tags?

Reply:
@mimisasouvanh this sounds like a great customer suggestion :)